lib/hanami/logger.rb

```diff
@@ -142,7 +146,7 @@ def _message_hash(message)
       # @since x.x.x
       # @api private
       def _format(hash)
-        hash.map { |k, v| "#{k}=#{v}" }.join(SEPARATOR)
+        hash.map { |k, v| "#{k}=#{v}" unless v.nil? }.join(SEPARATOR) + NEW_LINE
```

Comment:
@jodosha  yes, check this:
```
Puma starting in single mode...
* Version 3.4.0 (ruby 2.3.0-p0), codename: Owl Bowl Brawl
* Min threads: 0, max threads: 16
* Environment: development
* Listening on tcp://localhost:2300
Use Ctrl-C to stop
app=Web severity=DEBUG time=0.00774 addr=::1  now_time=01/Jun/2016:10:26:47 +0300 request_method=GET path=  http_version=HTTP/1.1 status=200
```

and this
```
Puma starting in single mode...
* Version 3.4.0 (ruby 2.3.0-p0), codename: Owl Bowl Brawl
* Min threads: 0, max threads: 16
* Environment: development
* Listening on tcp://localhost:2300
Use Ctrl-C to stop
app=Web severity=DEBUG time=0.005107 addr=::1 user= now_time=01/Jun/2016:10:27:13 +0300 request_method=GET path= query_string= http_version=HTTP/1.1 status=200 length=
```

Do you really what to see empty `length=`, `user=` and etc?